File: standalone/standalone_a/src/main/java/com/xia/fly/standalone/a/MainActivity.java
package com.xia.fly.standalone.a;

import com.xia.fly.module.a.ui.AFragment;
import com.xia.fly.ui.activities.ProxyActivity;
import com.xia.fly.ui.fragments.FlySupportFragment;

import androidx.annotation.NonNull;

public class MainActivity extends ProxyActivity {

    @NonNull
    @Override
    protected Class<? extends FlySupportFragment> setRootFragment() {
        return AFragment.class;
    }
}

File: standalone/standalone_a/src/main/java/com/xia/fly/standalone/a/MainActivity.kt
package com.xia.fly.standalone.a

import com.xia.fly.module.a.ui.AFragment
import com.xia.fly.mvp.BaseMvpPresenter
import com.xia.fly.mvp.BaseMvpView
import com.xia.fly.ui.activities.ProxyActivity
import com.xia.fly.ui.fragments.FlySupportFragment

/**
 * @author weixia
 */
class MainActivity<P : BaseMvpPresenter<BaseMvpView<P>>> : ProxyActivity<P>() {

    override fun setRootFragment(): Class<out FlySupportFragment<*>> {
        return AFragment::class.java
    }
}
